website/developer-docs/docs/writing-documentation.md

```diff
@@ -9,33 +9,43 @@ the documentation is easy to read and uses similar phrasing.
 
 Requirements:
 
--   Node 16 (or greater)
+-   Node.js 16 (or greater)
 
 The documentation site is situated in the `/website` folder of the authentik GitHub repository.
 
 The site is built using npm, below are some useful make commands:
 
--   Install: `make website-install` (Needed for any of the other tasks)
--   Formatting: `make website-lint-fix` or `make website` (Run this before committing)
--   Live editing: `make website-watch` (For real time viewing of changes)
+-   **Installation**: `make website-install`
 
+    This command is rquired before running any of the following commands, and after upgrading any dependencies.
+
+-   **Formatting**: `make website` or `make website-lint-fix`
+
+    Run this command before committing, otherwise the build will fail with an error about linting.
+
+-   **Live editing**: `make website-watch`
+
+    For real time viewing of changes, as you make them. This is a great way to verify that page formatting looks good.
+
+:::info
 Be sure to run the formatter before committing changes.
+:::
 
 ## General guidelines
 
--   authentik should always be stylized as `authentik` (with a lower-case a and ending with a k)
--   Documentation should use American english
--   Feel free to use Docusaurus-specific features, see [here](https://docusaurus.io/docs/next/markdown-features)
--   Use abbreviations where it makes sense (for commonly used terms like SAML and OAuth)
--   Phrasing should never blame the user, and should be subjective, i.e
-
-    -   **DON'T** `You may never click x.`
-    -   **DO** `x should never be clicked.`
+-   The product name authentik should always be stylized as `authentik` (with a lower-case "a" and ending with a "k").
+-   Documentation should use American English.
+-   Feel free to use [Docusaurus-specific features](https://docusaurus.io/docs/next/markdown-features)
```

Comment:
My only comment is that the invitation to use Docusaurus features could include (or inspire) someone to add MDX tags and features alongside the documentation, which could lead to the inclusion of malicious functionality in our documentation.  I don't know how they would do that-- I'm not generally a blackhat thinker-- but it's still something to contemplate.  Perhaps you should add a caveat that we only allow MDX features supplied by Docusaurus and no third-party MDX tags are allowed?

Reply:
I'll do some research on this. It encourages me to look very closely at any Docs PRs that include MDX.